openapi.yaml

```diff
@@ -0,0 +1,281 @@
+openapi: 3.0.3
+info:
+  title: Status Dashboard API
+  version: 1.0.0
+servers:
+  - url: http://localhost:8000
+tags:
+  - name: incidents
+    description: Incident management
+  - name: components
+    description: Operations about components
+paths:
+  /v1/components:
```

Comment:
I had thought `/api` was implemented elsewhere, but it's not.
So it has to be modified.